contracts/dapis/DapiServer.sol

```diff
@@ -572,23 +575,15 @@ contract DapiServer is
     /// a standard implementation of Airnode is being used, these can be
     /// expected to be correct. Either way, the assumption is that it does not
     /// matter for the purposes of a Beacon set update subscription.
-    /// @param subscriptionId Subscription ID
-    /// @param airnode Airnode address
-    /// @param relayer Relayer address
-    /// @param sponsor Sponsor address
-    /// @param timestamp Timestamp used in the signature
     /// @param data Fulfillment data (an `int256` encoded in contract ABI)
-    /// @param signature Subscription ID, timestamp, sponsor wallet address
-    /// (and fulfillment data if the relayer is not the Airnode) signed by the
-    /// Airnode wallet
```

Comment:
WDYT of keeping the references like this:
```
    /// @param // subscriptionId Subscription ID
    /// @param // airnode Airnode address
    /// @param // relayer Relayer address
    /// @param // sponsor Sponsor address
    /// @param // timestamp Timestamp used in the signature
    /// @param data Fulfillment data (an `int256` encoded in contract ABI)
    /// @param // signature Subscription ID, timestamp, sponsor wallet address
    /// (and fulfillment data if the relayer is not the Airnode) signed by the
    /// Airnode wallet
```

Reply:
Strongly agree, did this